src/System.Net.Http/src/System/Net/Http/HPack/DynamicTable.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace System.Net.Http.HPack
+{
+    internal class DynamicTable
+    {
+        private HeaderField[] _buffer;
+        private int _maxSize;
+        private int _size;
+        private int _count;
+        private int _insertIndex;
+        private int _removeIndex;
+
+        public DynamicTable(int maxSize)
+        {
+            _buffer = new HeaderField[maxSize / HeaderField.RfcOverhead];
```

Comment:
It looks like one of these is just constructed per connection, is that right?  So not worth pooling (at least not right now)?

Reply:
Yes.